### PR TITLE
implemented jax frontend function jax.numpy.imag and its unit tests

### LINKED ISSUE #13011: imag

### PR DESCRIPTION
**Pull request by Ajitofy**
Close #13011

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Ajitofy** [comment]
Plese assign someone to review this pr.. it's been more than 7 days I haven't received any review. Kindly look into it. @djl11

**Comment by Ajitofy** [comment]
Sure. Thanks @YushaArif99  for the detailed review.

**Comment by YushaArif99** [comment]
There also seem to be some merge conflicts. Can you kindly fix those as well 😅

**Comment by Ajitofy** [comment]
Ok sure @YushaArif99

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
